frameworks/hdfs/tests/config.py

```diff
@@ -20,58 +22,48 @@
 DOCKER_IMAGE_NAME = "mesosphere/hdfs-client:2.6.4"
 KERBERIZED_CLIENT_IMAGE_NAME = "nvaziri/kerberized-hdfs-client:dev"
 KERBERIZED_CLIENT_MEM_SIZE = 512
+KEYTAB = "hdfs.keytab"
 
-kerberized_hdfs_client_marathon_app = {
-    "id": "hdfsclient",
-    "mem": KERBERIZED_CLIENT_MEM_SIZE,
-    "container": {
-        "type": "MESOS",
-        "docker": {
-            "image": KERBERIZED_CLIENT_IMAGE_NAME,
-            "forcePullImage": True
-        },
-        "volumes": [
-            {
-                "containerPath": "/hadoop-2.6.0-cdh5.9.1/hdfs.keytab",
-                "secret": "hdfs_keytab"
-            }
-        ]
-    },
-    "secrets": {
-        "hdfs_keytab": {
-            "source": ""
-        }
-    },
-    "networks": [
-        {
-            "mode": "host"
-        }
-    ],
-    "env": {
-        "REALM": "",
-        "KDC_ADDRESS": "",
-        "JAVA_HOME": "/usr/lib/jvm/default-java",
-        "JVM_MaxHeapSize": KERBERIZED_CLIENT_MEM_SIZE
-    }
-}
+
+def get_kerberized_hdfs_client_app():
+    app_def_path = "{current_dir}/../tools/{client_id}".format(
+        current_dir=os.path.dirname(os.path.realpath(__file__)),
+        client_id="hdfsclient.json"
+    )
+    with open(app_def_path) as f:
+        app_def = json.load(f)
+
+    app_def["mem"] = KERBERIZED_CLIENT_MEM_SIZE
+    app_def["container"]["docker"]["image"] = KERBERIZED_CLIENT_IMAGE_NAME
+    app_def["container"]["volumes"][0]["containerPath"] = "/hadoop-2.6.0-cdh5.9.1/{}".format(KEYTAB)
+    return app_def
+
+
+def hdfs_write_command(filename, content_to_write):
```

Comment:
I generally prefer a `source, destination` in these kinds of arguments, but since utilities like `tar` swap these around this is also fine.